File: Order.java
import java.util.ArrayList;

public class Order {
    private String name;
    private ArrayList<Double> customerOrders;
    private double balance;
    private double[] menuPrices;

    public Order(String name, ArrayList<Double> customerOrders, double balance){
        this.name = name;
        this.customerOrders = customerOrders; //dont initialize already populated
        this.balance = balance;
    }

    public double totalPrice(){
        double totalPrice = 0;
        for(int i = 0; i<customerOrders.size(); i++ ){
            totalPrice += customerOrders.get(i);
        }
        return totalPrice;
    }

    public boolean validOrder(){
    if(balance - totalPrice() < 0 ) return false;
    else return true;
    }

    public double remainingBalance(){
        return balance-totalPrice();
    }

}

File: Order.py
class Order:
    def __init__(self, name, customer_orders, balance):
        self.name = name
        self.customer_orders = customer_orders
        self.balance = balance

    def valid_order(self):
        total_price = sum(self.customer_orders)
        return total_price <= self.balance

    def remaining_balance(self):
        total_price = sum(self.customer_orders)
        return self.balance - total_price

